src/main/java/org/opentripplanner/routing/api/request/request/JourneyRequest.java

```diff
@@ -1,6 +1,7 @@
 package org.opentripplanner.routing.api.request.request;
 
 import java.io.Serializable;
+import org.opentripplanner.routing.algorithm.raptoradapter.router.street.AccessEgressType;
```

Comment:
This import is not allowed, the enum is defined in the mapping between transitmodel and raptor and can only be used there. If we want to use this enum elsewhere we must create the enum in the domain model.

Reply:
Thanks. This wasn't really necessary so I removed the dependency